tests/integration-tests/tests/performance_tests/resources/bootstrap/metrics-publisher.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+# Cluster variables
+source /etc/parallelcluster/cfnconfig
+SHARED_DIR="$(echo $cfn_ebs_shared_dirs | cut -d ',' -f 1)"
+CLUSTER_NAME="${stack_name:?}"
+AWS_DEFAULT_REGION="${cfn_region:?}"
+
+# Load libraries
+source "${SHARED_DIR}/assets/lib/functions.sh"
+
+CW_NAMESPACE="ParallelCluster/PerformanceTests"
+CW_DIMENSIONS="ClusterName=${CLUSTER_NAME}"
+PCLUSTER="/usr/local/bin/pcluster"
+SQUEUE="/opt/slurm/bin/squeue"
+
+function publish_metric () {
+  local metric_name="$1"
+  local metric_value="$2"
+
+  put_metric ${AWS_DEFAULT_REGION} ${CW_NAMESPACE} "${CW_DIMENSIONS}" "${metric_name}" "${metric_value}"
```

Comment:
you missed: aws cloudwatch

Reply:
No, put_metric is a function in functions.sh